chef/cookbooks/bmc-nat/recipes/client.rb

```diff
@@ -27,6 +27,7 @@
 nat_address = nat_node[:crowbar][:network][:admin][:address]
 
 return if admin_subnet == bmc_subnet && admin_netmask == bmc_netmask
+return if node[:platform_family] == "windows"
```

Comment:
We usually put the check at the top of the recipe (see logging::client). Can you do the small change (and backport to 3.0)?